### ISSUE TITLE
[Bug]: 0.6.86 is latest version that support reverse proxying at a path that is not `/`

### ISSUE THREAD

**Issue by sjorge** [nurikk/z2m-frontend#1495]
### What happened?

After v0.6.86 reverse proxying on path that is not `/` no longer works.

It seems all asset URLs are now generate with `/assets/...` as absolute path.

This is easy to see by checking out the generate content for the index between v0.6.86 and  the latest version and looking at the href's present.

There is a painful workaround of using mod_perl to rewrite all of them but that is horrible.

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari

### Relevant stacktrace

```shell
n/a
```

### Did you download state.json.zip?

I will not attach state.json.zip

**Comment by sjorge** [comment]
I managed to fix it in a not so ugly way as mod_perl
```
<Location /zigbee2mqtt/>
 AddOutputFilterByType INFLATE;SUBSTITUTE;DEFLATE text/html
 AddOutputFilterByType INFLATE;SUBSTITUTE;DEFLATE text/css
 AddOutputFilterByType INFLATE;SUBSTITUTE;DEFLATE application/javascript
 Substitute "s|/assets/|/zigbee2mqtt/assets/|"
</Location>
```

Where the /zigbee2mqtt/ is the location where I proxy to the frontend from.

I'm closing it for now, I'm not sure how common it is for people to proxy from a URI instead of a full vhost. I might be the only one with this usecase.

**Comment by nurikk** [comment]
What's the problem exactly? I can tune assets bundler

**Comment by sjorge** [comment]
Paths in src, href and other fields are now generated with an absolute path starting with /assets before they used to be relative paths.

With absolute paths the browser will always try to load them from domain.tld/assets/…

Even if they base path after proxying is something else like domain.tld/z2m/

**Comment by nurikk** [comment]
Hi, try to switch to the latest dev branch

https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html#switch-to-the-dev-branch

**Comment by nurikk** [comment]
Should work fine in latest dev with frontend 0.6.96

**Comment by sjorge** [comment]
Fixed in 0.6.96!
